src/orderbooks.md

```diff
@@ -0,0 +1,204 @@
+
+By leveraging the power of blockchain technology, order books on decentralized exchanges offer several advantages:
+
+- **Better price understanding**: Buyers and sellers will be able to see actual offers, enabling them to make more informed decisions about the true market value of assets. This can lead to more efficient trading.
+- **Improved liquidity**: Buyers and sellers will have greater control over the prices at which they trade. This helps attract more market participants, leading to increased liquidity and a more dynamic market.
+- **Additional liquidity source for the Liquidity Proxy**: Traders will have an alternative option to use the Liquidity Proxy, which can positively impact trading outcomes.
+- **Reduced slippage**: In XYK markets, large trades can cause significant price changes due to the trade size impacting the market price. With order books, traders can place limit orders at the desired price, reducing the risk of slippage.
+- **More trading options**: Traders will have access to various trading mechanisms, including market orders, limit orders, stop orders, and more. This can lead to more sophisticated trading strategies and better risk management.
+- **Alternative market for NFT trading**: NFTs can have their own order book, where users can place buy and sell orders. While it may not be the optimal mechanism for NFT trading due to their uniqueness and the need for customized volume and price steps (auction models or peer-to-peer mechanisms where buyers and sellers can directly negotiate the price of NFTs are traditionally considered more suitable), the order book mechanism can be more relevant for NFT trading compared to Pool XYK.
+- **Other benefits inherited from DEX characteristics**:
+  - _Permissionless accessibility_: DEXs allow anyone to participate in trading digital assets without a centralized intermediary. This promotes financial inclusion and provides access to a global user base, regardless of geographical location.
+  - _Enhanced ownership and control_: All assets remain under the ownership and control of users.
+  - _Transparency_: Users can verify authenticity, ownership history, and transaction data, fostering trust and reducing the risk of fraud or manipulation by a centralized intermediary.
+  - _Decentralization benefits_: Reduced risks of hacks, downtime, or data breaches associated with centralized platforms.
+  - _Community-driven development_: Users can create order books themselves, contribute to liquidity, set their own trading parameters, and influence development using decentralized governance tools. This fosters innovation, competition, and a vibrant ecosystem.
+
+While order books on decentralized exchanges offer numerous benefits, it's important to consider their limitations:
+
+- **Scalability**: Blockchains have limitations in processing transaction volumes.
+- **Liquidity**: Liquidity of decentralized order books can be limited.
```

Comment:
```suggestion
```
This is too general. Liquidity can be limited on centralised exchanges as well, and can be deeper in other decentralised exchanges.
Does not bring any info / value to readers.